PROPOSAL.md

```diff
@@ -0,0 +1,70 @@
+# COMP 4350 Proposal Team - Spendr
+
+## Project Summary and Vision
+
+Spendr is designed to simplify group expenses by offering a comprehensive platform for users to collaboratively track and manage shared expenses, ensuring transparency, accuracy, and ease of settlements.
+
+## Core Features
+
+1. User Management
+2. Group Management
+3. Expense Categorization
+4. Expense Analytics
```

Comment:
Are we doing expense categorization or ai parser?

Reply:
Oops, I forgot to adjust the core features!

Fixed in https://github.com/BarbzCodez/Spendr/pull/1/commits/4a8c59dbf68207357d49670b822f5007f1ebeafe